submissions/nadsatt/friends-app/modules/page-link-list.js

```diff
@@ -0,0 +1,151 @@
+import { PageLink, CornerPageLink } from './page-link-items.js';
+
+export class PageLinkList {
+    constructor(userService, userCardList){
+        this.userService = userService;
+        this.userCardList = userCardList;
+        this.fragment = new DocumentFragment();
+
+        this.usersPerPage = 10;
+        this.maxPageLinksNumber = 5;
+
+        this.defineCornerPageLinks();
+        this.defineElement();
+        this.insertElement();
+        this.defineElementClickHandler();
+    }
+
+    defineCornerPageLinks(){
+        this.leftPageLinks = ['First', 'Prev'].map(value => new CornerPageLink(value).element);
+        this.rightPageLinks = ['Next', 'Last'].map(value => new CornerPageLink(value).element);
```

Comment:
You can save instance of classes instead of elements and then call method of instances itself.

Why get instances thru DOM elements?